torchvision/transforms/functional.py

```diff
@@ -855,3 +857,45 @@ def erase(img, i, j, h, w, v, inplace=False):
 
     img[:, i:i + h, j:j + w] = v
     return img
+
+def elastic_transform(img, alpha, sigma, random_state):
+    """Elastic deformation of image as described in [Simard2003]_.
+    .. [Simard2003] Simard, Steinkraus and Platt, "Best Practices for
+       Convolutional Neural Networks applied to Visual Document Analysis", in
+       Proc. of the International Conference on Document Analysis and
+       Recognition, 2003.
+
+    Args:
+        img (PIL Image): Image on which elastic transformation is to be applied
+        alpha (float): Scaling factor as described in [Simard2003]
+        sigma (float): Elasticity coefficient as described in [Simard2003]
+        random_state (int): Random state to initialize the Gaussian kernel
+
+    Returns:
+        PIL Image: Image with elastic distortions.
+    """
+    if not _is_pil_image(img):
+        raise TypeError('img should be PIL Image. Got {}'.format(type(img)))
+
+    img = np.asarray(img)
```

Comment:
This conversion dismisses that the PIL image has different modes and has to be scaled and converted accordingly. You could use `img = to_tensor(img).numpy()` if you don't want to handle the correct conversion yourself.

Reply:
Yes, thank you for pointing this out. That'd definitely be the better way to convert the image into `numpy`.